plugins/ar_spectral.py

```
@@ -1,1098 +1,1099 @@
+            yres = 1
+        return int(xres), int(yres)
+
+    def _estimateCCDTime(self):
+        """
+        Estimate the time it will take for measurement at one ebeam position
+        """
+        # Each pixel = the exposure time (of the detector) + readout time +
+        # 10ms overhead + 20% overhead
+
+        binning = (self.binninghorz.value, self.binningvert.value)
+        res = [s / b for s, b in zip(self._detector.resolution.range[1], binning)]
```

Ruff (0.15.9)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)